src/services/sync/strategies/LDAPSyncer.js

```diff
@@ -44,47 +47,62 @@ class LDAPSyncer extends SystemSyncer {
 
 	getCurrentYearAndFederalState() {
 		return Promise.all([
-			this.app.service('years').find({ $orderby: { name: -1 } }),
+			this.app.service('years').find(),
 			this.app.service('federalStates').find({ query: { abbreviation: 'NI' } }),
 		]).then(([years, states]) => {
-			if (years.total == 0 || states.total == 0) {
-				return Promise.reject('Database should contain at least one year and one valid federal state');
+			if (years.total === 0 || states.total === 0) {
+				return Promise.reject(
+					new Error('Database should contain at least one year and one valid federal state'),
+				);
 			}
-			return Promise.resolve({ currentYear: years.data[0]._id, federalState: states.data[0]._id });
+			const currentYear = new SchoolYearFacade(years).defaultYear;
+			return Promise.resolve({ currentYear, federalState: states.data[0]._id });
```

Comment:
yes. For now, every school created via LDAP is in one state. Since nobody creates schools via LDAP (this was a feature in the early days of n21), it's low priority to fix this. (a ticket already exists)